### ISSUE TITLE
Have marvin send the most recent statuses for a channel

### ISSUE THREAD

**Issue by jakerella** [jamesdabbs/lita-panic#4]
I would like to ask marvin for a summary of the most recent scores for a channel using a direct message. This would allow me to see at a glance how everyone is. While this does happen once the survey is complete... my channel rarely completes. Here is an example of the interaction I would like to see:

```
jordan: status #frontend_feb2017
marvin: here is the most recent status results for #frontend_feb2017...
  john: 3
  jill: 4
  james: (none)
  jane: 5
```

**Comment by jakerella** [comment]
@rposborne @rposborne @rposborne @rposborne @rposborne @rposborne @rposborne @rposborne @rposborne @rposborne @rposborne @rposborne @rposborne @rposborne @rposborne @rposborne @rposborne @rposborne @rposborne @rposborne @rposborne @rposborne @rposborne @rposborne @rposborne 🍆 @rposborne @rposborne @rposborne @rposborne @rposborne @rposborne @rposborne @rposborne @rposborne @rposborne @rposborne @rposborne @rposborne @rposborne @rposborne @rposborne @rposborne @rposborne @rposborne @rposborne @rposborne @rposborne

**Comment by rposborne** [comment]
@jakerella This should be solved by #5 you can look it over and push it to our marvin.

**Comment by jakerella** [comment]
Reopen due to issue we saw?
